haiku-apps/beae/beae-1.2.recipe

```diff
@@ -4,9 +4,11 @@ in/out, analyzing, etc."
 HOMEPAGE="https://github.com/HaikuArchives/BeAE"
 COPYRIGHT="2003 Xentronix"
 LICENSE="BSD (3-clause)"
-REVISION="3"
-SOURCE_URI="$(HOMEPAGE)/archive/a90a9872420afe1a06537c798bfdfdc36d631323.tar.gz"
-SOURCE_DIR="beae-1.2"
+REVISION="4"
+SOURCE_URI="https://github.com/HaikuArchives/BeAE/archive/v1.2.tar.gz"
+CHECKSUM_SHA256="d69a16821f86eda6e94b8297928f9def75f5c750e1e7d6b946d0b195fdc88943"
+SOURCE_DIR="BeAE-1.2"
```

Comment:
It might be a good idea to replace
```
SOURCE_DIR="BeAE-1.2"
```
by
```
SOURCE_DIR="BeAE-$portVersion"
SOURCE_FILENAME="BeAE-$portVersion.tar.gz"
```
(and also replace "v1.2.tar.gz" by "v$portVersion.tar.gz" in SOURCE_URI.)